content/en/users/compute/content-distribution/_index.md

```diff
@@ -55,26 +55,39 @@ All VO content managers should join the CVMFS-UPLOADER-USERS mailing list in
 
 ## Distributing new content
 
-To login to the service, make sure you have a valid X509 proxy (with the same DN
-provided [in this step](#requesting-access)), and execute the following command:
+To login to the service, just ssh:  
 
 ```shell
-$ gsissh -p 1975 cvmfs-upload01.gridpp.rl.ac.uk
+$  ssh @cvmfs-upload01.gridpp.rl.ac.uk
```

Comment:
```suggestion
$  ssh cvmfs-upload01.gridpp.rl.ac.uk
```

Reply:
Only method currently provided is interactive login.
Web access is not [yet] implemented. 
I know some people have a crontab in their repository to git pull periodically. That is OK, but needs to be requested and approved by our security team.

Reply:
I still see the `@` in the command line there, this fails to run